filament/backend/include/backend/DriverEnums.h

```diff
@@ -219,42 +212,45 @@ enum class ElementType : uint8_t {
     HALF4,
 };
 
+//! Face culling Mode
 enum class CullingMode : uint8_t {
-    NONE,
-    FRONT,
-    BACK,
-    FRONT_AND_BACK
+    NONE,               //!< No culling, front and back faces are visible
+    FRONT,              //!< Front face culling, only back faces are visible
+    BACK,               //!< Back face culling, only front faces are visible
+    FRONT_AND_BACK      //!< Front and Back, geometry is not visible
 };
 
-//! PixelDataFormat
+//! Pixel Data Format
 enum class PixelDataFormat : uint8_t {
-    R,
-    R_INTEGER,
-    RG,
-    RG_INTEGER,
-    RGB,
-    RGB_INTEGER,
-    RGBA,
-    RGBA_INTEGER,
-    UNUSED,                 // used to be rgbm
-    DEPTH_COMPONENT,
-    DEPTH_STENCIL,
-    ALPHA
-};
-
+    R,                  //!< One Red channel, float
```

Comment:
Super minor but I would've simply said `Four channels` instead of `Four Red, Green, Blue and Alpha channels` etc, since you could put arbitrary data in there.

Reply:
You can put arbitrary data, but calling them red,green,etc... tells you in which component of the vec4 they appear on the shader side. e.g. `R` v.s. `ALPHA`, also how it can influence blending,  for render targets.